src/test/java/com/vackosar/gitflowincrementalbuild/boundary/UnchangedProjectsRemoverTest.java

```diff
@@ -526,6 +526,28 @@ public void singleChanged_forceBuildModules() throws GitAPIException, IOExceptio
         verify(mavenSessionMock).setProjects(Arrays.asList(moduleA, changedModuleMock));
     }
 
+    @Test
+    public void singleChanged_forceBuildModules_conditionalMatches() throws GitAPIException, IOException {
+        MavenProject changedModuleMock = addModuleMock(AID_MODULE_B, true);
+
+        addGibProperty(Property.forceBuildModules, changedModuleMock.getArtifactId() + "=" + moduleA.getArtifactId());
```

Comment:
I think that this test is the wrong way around.
A is the condition whereas B is the module to be forced.

Reply:
This test tries to mimic the change status of a module. And without a changed module there will be forced module.
Also, every module mock created via `addModuleMock()` has module A as an upstream dependency. Module a is not marked changed by default.
So, this test mimics that module B is changed and that a config exists that says: build A forcibly when B changed (`module-B=module-A`).
LGTM?

Reply:
Yes, apologies - it looks like it corresponds to the description in the readme at https://github.com/gitflow-incremental-builder/gitflow-incremental-builder/pull/424/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R626-R629

I think that I misread the original proposal. Thanks.

Reply:
No worries! If you think it makes more sense the other way around then juts let me know. I actually derived that solution from your initial proposal. 🙂